lib/rspec-puppet-facts.rb

```diff
@@ -179,8 +179,9 @@ def on_supported_os_implementation(opts = {})
     os_facts_hash
   end
 
+  # @api private
   def stringify_keys(hash)
```

Comment:
do you know if that's used somewhere? should we mark it as breaking change? (I want to drop old rubies anyways)

Reply:
I think it should only be internal. And marking it with a comment doesn't mean it can't be called anymore, it's only a signal people consuming the API.